thredded.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://www.thredded.com'
   s.summary     = 'A messageboard engine'
   s.license     = 'MIT'
-  s.description = 'A messageboard engine for Rails 4.0 apps'
+  s.description = 'A messageboard engine for Rails 4.0+ apps'
```

Comment:
This should be 4.1+, and the Rails dependency below at 4.1+ as well, because db_text_search doesn't actually support 4.0. 4.0 also no longer receives bugfixes, so nobody should be using it.